src/main/webapp/app/components/oncokbTable/OncoKBTable.tsx

```diff
@@ -47,20 +48,37 @@ export default class OncoKBTable<T> extends React.Component<
     });
   }
 
+  SearchBar = () => {
+    return this.props.disableSearch ? (
+      <></>
+    ) : (
+      <div className="d-flex">
+        <div className="ml-auto">
+          <input
+            onChange={(event: any) => {
+              this.searchKeyword = event.target.value.toLowerCase();
+            }}
+            className="form-control input-sm"
+            type="text"
+            placeholder="Search ..."
+          />
+        </div>
+      </div>
+    );
+  };
+
   render() {
     return (
       <div>
-        {this.props.disableSearch ? undefined : (
-          <div className="d-flex">
-            <div className="ml-auto">
-              <input
-                onChange={(event: any) => {
-                  this.searchKeyword = event.target.value.toLowerCase();
-                }}
-                className="form-control input-sm"
-                type="text"
-                placeholder="Search ..."
-              />
+        {this.props.filters === undefined ? (
+          <this.SearchBar />
```

Comment:
I think we need to consider here is that the div with row className should always be there. If the filters is not null, then render the filters. If the search is enabled, then render the search bar.